src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -378,6 +378,30 @@
     <value>into clause</value>
     <comment>{Locked="into"} "into" is a C# keyword and should not be localized.</comment>
   </data>
+  <data name="is_pattern" xml:space="preserve">
+    <value>is pattern</value>
+    <comment>{Locked="is"} "is" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="deconstruction" xml:space="preserve">
+    <value>deconstruction</value>
+  </data>
+  <data name="tuple" xml:space="preserve">
+    <value>tuple</value>
+  </data>
+  <data name="local_function" xml:space="preserve">
+    <value>local function</value>
+  </data>
+  <data name="out_var" xml:space="preserve">
+    <value>out variable</value>
+    <comment>{Locked="out"} "out" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="ref_local_or_expression" xml:space="preserve">
+    <value>ref local or expression</value>
+    <comment>{Locked="ref"} "ref" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="v7_switch" xml:space="preserve">
+    <value>v7 switch</value>
```

Comment:
Updated to "C# 7 enhanced switch statement".
When I tried EnC in VS, I was not able to observe the error message or the diagnostics, except for an error popup with a generic message (there was an error with this edit & continue) and squiggles on the span. I didn't see anything in the error or output panes.
Do you know if the error message appears somewhere else?